data/recipes/gcp_forensics.json

```diff
@@ -11,22 +11,25 @@
           "remote_instance_name": "@instance",
           "incident_id": "@incident_id",
           "zone": "@zone",
+          "create_analysis_vm": "@create_analysis_vm",
           "disk_names": "@disks",
           "all_disks": "@all_disks",
           "boot_disk_size": "@boot_disk_size",
           "boot_disk_type": "@boot_disk_type",
-          "cpu_cores": 16,
+          "cpu_cores": "@cpu_cores",
           "image_project": "ubuntu-os-cloud",
           "image_family": "ubuntu-1804-lts"
       }
   }],
   "args": [
       ["remote_project_name", "Name of the project containing the instance / disks to copy ", null],
-      ["incident_id", "Incident ID to label the VM with.", null],
+      ["--analysis_project_name", "Name of the project where the analysis VM will be created", null],
+      ["--incident_id", "Incident ID to label the VM with.", null],
       ["--instance", "Name of the instance to analyze.", null],
       ["--disks", "Comma-separated list of disks to copy.", null],
       ["--all_disks", "Copy all disks in the designated instance. Overrides disk_names if specified", false],
-      ["--analysis_project_name", "Name of the project where the analysis VM will be created", null],
+      ["--create_analysis_vm", "Create an analysis VM.", true],
+      ["--cpu_cores", "Number of CPU cores of the analysis VM", 4],
```

Comment:
Someone specifying this parameter through the CLI will turn it into an `str`. Please make sure this is converted to an int during `SetUp`

Reply:
Actually, this should be changed in dftimewolf_recipes.py:83. I openeed https://github.com/log2timeline/dftimewolf/issues/341 for that. (but in the meantime, please make sure it's converted to an int)

Reply:
done in setup method